src/test/java/Homework18.java

```diff
@@ -8,11 +8,12 @@
 
 import java.time.Duration;
 
-public class Homework18 extends BaseTest {
+public class Homework18 extends BaseTest{
 
     @Test
     public static void playSongTest() throws InterruptedException {
 
+
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/210978111-ddff0fa6-f5bb-49ed-8e4e-09682ba8686b.png)

![image](https://user-images.githubusercontent.com/104761482/210978141-764002e6-9a64-48e4-a8cd-6f526e197c40.png)

Hint

```
        WebElement equalizerBars = driver.findElement(By.cssSelector("[alt='Sound bars']"));
        Thread.sleep(5000);
        Assert.assertTrue(equalizerBars.isDisplayed());
```

![image](https://user-images.githubusercontent.com/104761482/210978483-46fb7d4f-3d55-400b-92fa-f2aa6a5beb3a.png)

Reply:
Thanks! That's weird because everything passed for me.

![image](https://user-images.githubusercontent.com/121219634/211128683-e05eadc0-26aa-4225-b4a5-6a6693c174ae.png)

Reply:
![image](https://user-images.githubusercontent.com/121219634/211128778-028357fe-c5bc-4e2f-926e-9ee0a0d868f8.png)